### DISCUSSION TITLE
Feature request: `separateMultipleMinor` option

### DISCUSSION THREAD

**Discussion by ryota-ka** [renovatebot/renovate#20963] [Request Help] [answered]
### Type of discussion.

I'm proposing an idea

### Tell us more.

IIUC Renovate has the `separateMultipleMajor` option to mitigate the migration path of breaking changes, by updating multiple major versions one by one.

Unfortunately, not all the libraries don't follow the Semantic Versioning 2.0.
Despite it is distributed through npm, [TypeScript does not adhere to SemVer](https://github.com/microsoft/TypeScript/issues/14116), thus any minor version bump can introduce a breaking change.

Would it be nice if there were the `separateMultipleMinor` option so that we can mitigate the migration of (for example) TypeScript versions?
What do you think about it?

**Comment by rarkins** [comment, marked as answer]
- #6576

**Reply by ryota-ka** [reply]
Thank you @rarkins!
I've searched the similar request only in discussions so I seem to have missed the issue.
